parsl/app/bash.py

```diff
@@ -77,10 +77,10 @@ def open_std_fd(fdname):
         returncode = proc.returncode
 
     except subprocess.TimeoutExpired:
-        raise pe.AppTimeout("[{}] App exceeded walltime: {} seconds".format(func_name, timeout))
+        raise pe.AppTimeout("App {} exceeded walltime: {} seconds".format(func_name, timeout))
 
     except Exception as e:
-        raise pe.AppException("[{}] App caught exception with returncode: {}".format(func_name, returncode), e)
+        raise pe.AppException("App {} caught exception with returncode: {}".format(func_name, returncode), e)
```

Comment:
Heh.  At some point, every project gets to "wrestle" with the accepted format for messaging in (opaque) strings.

My only suggestion here: consider moving to f-strings as the other hunk in this PR did.